### ISSUE TITLE
[idea] load directly from ejson

### ISSUE THREAD

**Issue by burke** [Shopify/shadowenv#125]
This would require some unfortunate linking or rewriting but it would be kind of excellent to have a single `(env/load-from-ejson "config/secrets.ejson" ".env")` or something.

Challenges:
* Would require an additional change detection feature
* A bit more expensive than current evaluation
* New failure modes (key not present, key invalid, ciphertext invalid, JSON invalid, file missing)
* ejson is written in go so we couldn't trivially import it as a lib, and I don't love the idea of shelling out.

Benefits:
* Disincentivizes writing decrypted secrets to disk
* Appealingly nice integration between our tools.

**Comment by tobi** [comment]
that being said, we should also disincentive having decrypted secrets in ENV as env can leak in debug messages (has happened before). I think having the encrypted parts in env and having easy decription client libraries available would be good. 

this may be too much of a change but have we considered age/rage here? Very very good tools topically related to this.

**Comment by burke** [comment]
Yeah, that's a good point, we should instead at least try to normalize a pattern that leaves them encrypted until required.
